packages/xsnap/package.json

```diff
@@ -50,7 +50,14 @@
     "LICENSE*",
     "api.js",
     "build.env",
-    "src"
+    "moddable/modules/data",
+    "moddable/xs/includes",
+    "moddable/xs/makefiles",
+    "moddable/xs/platforms/*.h",
+    "moddable/xs/sources",
+    "src",
+    "xsnap-native/xsnap/makefiles",
+    "xsnap-native/xsnap/sources"
```

Comment:
This is the bit that makes it unnecessary for the `postinstall` script to clone `git` submodules.